src/mbgl/actor/mailbox.cpp

```diff
@@ -22,6 +25,28 @@ void Mailbox::close() {
     closed = true;
 }
 
+void Mailbox::activate(Scheduler& scheduler_) {
```

Comment:
Why not `open`, for parallelism with `close`?

Reply:
That's what I was thinking originally, but I think it's misleading: when a mailbox is `closed()` sending messages to it is, by design, a noop.  When it's not yet active, messages get queued.  Using open/closed, but having "closed as not-yet-open" and "closed as previously opened" behave differently seems confusing